package/lib/compileFunctions.js

```diff
@@ -57,6 +58,17 @@ module.exports = {
         });
       }
 
+      if (funcObject.vpcEgress) {
```

Comment:
Why are we applying this only if `vpcEgress` is set directly on function? What is the purpose of `provider` level setting then? It will never be effective as it will be always overriden by value set on function or not applied at all

Reply:
Hey! sorry, I'm not following you here. What do you mean that the value will be overwritten?

I have the following configuration ([https://i.imgur.com/7kENrcp.png](url)) in my `serverless.yml` file. I think `vpcEgress` is configured at the function level.

I deployed using this fork and got the following result:

[https://i.imgur.com/tb748wY.png](url)

I wait your answer, thank you!

Reply:
What I mean here is that while we support `vpcEgress` setting on `provider` level, it will never take effect because the logic is applied only when the setting is applied directly on function. I would assume that if it's only set on `provider` level, then it should be applied to all functions in service, correct?

Reply:
Hey! i think i got it, i just applied the change to make it through the provider level.
do you think it's ok?
thanks!